src/Process/ObjectRemovalHelper.js

```diff
@@ -1,3 +1,14 @@
+function disposeSingleMaterialAndTextures(material) {
+    material.dispose();
+    // dispose textures
+    for (const key of Object.keys(material)) {
+        const val = material[key];
+        if (val && val.isTexture) {
+            val.dispose();
+        }
+    }
+}
+
```

Comment:
what type of materials do you want to dispose?

Reply:
I made it generic so all type of materials and their textures can be disposed with this method